test/common/operation.go

```diff
@@ -46,7 +47,7 @@ func (client *Client) SendFakeEventToAddressable(
 	senderName,
 	addressableName string,
 	typemeta *metav1.TypeMeta,
-	event *base.CloudEvent,
+	event *resources.CloudEvent,
```

Comment:
Seems weird to have CloudEvent in resources as it's not an actual knative nor k8s resource. Maybe leave that one in base?

Reply:
There will be an import loop if I move it to `base` package. 
I created a new file under `resources` and moved it there. It's not a kind of resource, but something used by other resources, so I think it makes sense to have it there.